operations/mimir-mixin/dashboards/writes.libsonnet

```diff
@@ -137,6 +137,27 @@ local filename = 'mimir-writes.json';
         )
       )
     )
+    .addRowIf(
+      $._config.forwarding_enabled,
+      $.row('Distributor Forwarding')
+      .addPanel(
+        $.panel('Requests / sec') +
+        $.successFailurePanel(
+          'sum(rate(cortex_distributor_forward_requests_total{%(distributorMatcher)s}[$__rate_interval]))' % $.jobMatcher($._config.job_names.distributor),
+          'sum(rate(cortex_distributor_forward_errors_total{%(distributorMatcher)s}[$__rate_interval]))' % $.jobMatcher($._config.job_names.distributor)
+        )
+      )
+      .addPanel(
+        $.panel('Latency') +
+        $.latencyPanel('cortex_distributor_forward_requests_latency_seconds', '{%s}' % $.jobMatcher($._config.job_names.distributor))
+      )
+      .addPanel(
+        $.timeseriesPanel('Per %s p99 latency' % $._config.per_instance_label) +
+        $.hiddenLegendQueryPanel(
+          'histogram_quantile(0.99, sum by(le, %s) (rate(cortex_distributor_forward_requests_latency_seconds_bucket{%s}[$__rate_interval])))' % [$._config.per_instance_label, $.jobMatcher($._config.job_names.distributor)], ''
```

Comment:
Why empty legend? Ok the legend is hidden, but it's also displayed in the tooltop. I think you want `{{ %s }} % $._config.per_instance_label` for the legend.

Reply:
thx, you're right, I added it: 14a541fad6